manifests/jetty.pp

```diff
@@ -110,6 +106,13 @@
     mode    => '0644',
     recurse => false,
     notify  => Class['shibboleth_idp::service'],
+  } ->
+
+  file { $idp_jetty_log_dir:
+    ensure => directory,
+    owner  => $shibboleth_idp::shib_user,
+    group  => $shibboleth_idp::shib_group,
+    mode   => '0644',
```

Comment:
I forget to think about (file vs directory) + recursive or not situations. I've changed these.

I do get frustrated when I can't read log files, but if we retain sudo, that's not an issue. I'd also like to see the shib logs get into ELK at some point.